### ISSUE TITLE
JAWS reader does not announce the label when a MultiSelect is wrapped by a label element

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#2935]
### Bug report
JAWS reader does not announce the label when a MultiSelect is wrapped by a label element.

### Reproduction of the problem
Issue could be reproduced in [the following Dojo](http://dojo.telerik.com/UqeFU/13). The label on the Simple select element is correctly announced.

**Comment by jivanova** [comment]
Fix included in https://github.com/telerik/kendo/commit/8d44233256efbb392b48896cf0ecb4d3fa7257a0. Note that placing the label wrapping the select element might cause repetition of the label by a Screen Reader. The issue stems from the MultiSelect rendering that is inserted over the select used for initialization and the label should not be its parent. We recommend configuring label outside the MultiSelect:
``` html
<label for="multiselectReadonly">Volvo
</label>
<select id="multiselectReadonly" multiple="multiple" style="width:100%">
    <option>Item1</option>
    <option>Item2</option>
    <option>Item3</option>
    <option>Item4</option>
</select>
```
